pkg/collectsub/server/server.go

```diff
@@ -126,6 +127,7 @@ func (s *server) Serve(ctx context.Context) error {
 				grpc_zap.UnaryServerInterceptor(logger.Desugar()),
 				contextToZapFieldsUnaryServerInterceptor(),
 			)),
+		grpc.MaxRecvMsgSize(16777216),
```

Comment:
depending on how often we see this changing, we could make it an env variable for now with a default value.